docs/OV_Runtime_UG/multi_device.md

```diff
@@ -160,6 +156,12 @@ Additional Resources
     <iframe allowfullscreen mozallowfullscreen msallowfullscreen oallowfullscreen webkitallowfullscreen width="560" height="315" src="https://www.youtube.com/embed/xbORYFEmrqU" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 
-.. note:: This video is currently available only for C++, but many of the same concepts apply to Python.
```

Comment:
Do you think the video is still useful? Maybe we should consider removing it so that it does not misinform OV users.
We are planning on updating many videos from the old playlist but since AUTO is slowly "replacing" MULTI, this video is unlikely to be refreshed.

Reply:
Removed it to avoid the confusion.